<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>751</width>
    <height>426</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>212</red>
        <green>230</green>
        <blue>202</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>212</red>
        <green>230</green>
        <blue>202</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>212</red>
        <green>230</green>
        <blue>202</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>212</red>
        <green>230</green>
        <blue>202</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>212</red>
        <green>230</green>
        <blue>202</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>212</red>
        <green>230</green>
        <blue>202</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>212</red>
        <green>230</green>
        <blue>202</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>212</red>
        <green>230</green>
        <blue>202</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>212</red>
        <green>230</green>
        <blue>202</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>212</red>
         <green>230</green>
         <blue>202</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>212</red>
         <green>230</green>
         <blue>202</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>212</red>
         <green>230</green>
         <blue>202</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>212</red>
         <green>230</green>
         <blue>202</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>212</red>
         <green>230</green>
         <blue>202</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>212</red>
         <green>230</green>
         <blue>202</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>212</red>
         <green>230</green>
         <blue>202</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>212</red>
         <green>230</green>
         <blue>202</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>212</red>
         <green>230</green>
         <blue>202</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <widget class="QLabel" name="TestNum">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>60</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>检测数100</string>
    </property>
   </widget>
   <widget class="QLabel" name="PassNum">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>60</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-right-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>通过数100</string>
    </property>
   </widget>
   <widget class="QLabel" name="NoPassNum">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>60</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>未通过数100</string>
    </property>
   </widget>
   <widget class="QLabel" name="speed">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>60</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>皮带速度显示2m/s</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SetZero">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>55</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(83, 148, 160);
color: rgb(255, 250, 240);
border-radius:5px;</string>
    </property>
    <property name="text">
     <string>计数清零</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Management">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>390</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(83, 148, 160);
color: rgb(255, 250, 240);
border-radius:5px;</string>
    </property>
    <property name="text">
     <string>产品管理</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Sensitivity">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>390</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(83, 148, 160);
color: rgb(255, 250, 240);
border-radius:5px;</string>
    </property>
    <property name="text">
     <string>灵敏度</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Phase">
    <property name="geometry">
     <rect>
      <x>346</x>
      <y>390</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(83, 148, 160);
color: rgb(255, 250, 240);
border-radius:5px;</string>
    </property>
    <property name="text">
     <string>相位</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Log">
    <property name="geometry">
     <rect>
      <x>504</x>
      <y>390</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(83, 148, 160);
color: rgb(255, 250, 240);
border-radius:5px;</string>
    </property>
    <property name="text">
     <string>日志</string>
    </property>
   </widget>
   <widget class="QPushButton" name="System_set">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>390</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(83, 148, 160);
color: rgb(255, 250, 240);
border-radius:5px;</string>
    </property>
    <property name="text">
     <string>系统设置</string>
    </property>
   </widget>
   <widget class="QLabel" name="DateTime">
    <property name="geometry">
     <rect>
      <x>567</x>
      <y>10</y>
      <width>179</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(33, 28, 23);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>2017/02/01  13:28</string>
    </property>
   </widget>
   <widget class="QLabel" name="SpeedDisLabel">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(33, 28, 23);</string>
    </property>
    <property name="text">
     <string>输送带速度显示</string>
    </property>
   </widget>
   <widget class="QLabel" name="Detect">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(33, 28, 23);</string>
    </property>
    <property name="text">
     <string>实时检测</string>
    </property>
   </widget>
   <widget class="QLabel" name="ProNum">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>产品编号</string>
    </property>
   </widget>
   <widget class="QLabel" name="ProName">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>产品名称</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Jur_And_User">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(83, 148, 160);
color: rgb(255, 250, 240);
border-radius:5px;</string>
    </property>
    <property name="text">
     <string>权限/用户名</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
